pages/issues/issues.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Helmet } from "react-helmet";
+import { Link } from 'react-router';
+import IssueSelector from '../../components/issue-selector/issue-selector.jsx';
+import Service from '../../js/service';
+import Utility from '../../js/utility';
+
+class Issues extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      issues: []
+    };
+  }
+
+  componentDidMount() {
+    this.getIssues();
+  }
+
+  getIssues() {
+    Service.issues
+      .get()
+      .then((issues) => {
+        // sort issues alphabetically by their name
+        issues.sort((a,b) => {
```

Comment:
Ahhhh I now recall why I didn't go with `issues.sort()`. `issues` should be an array of issue object `{name: ``, description: ``}` and not just an array of issue name strings.